### ISSUE TITLE
AcqKnowledge graph file creation - calibration is not clear

### ISSUE THREAD

**Issue by oesteban** [TheAxonLab/hcph-sops#120]
There are several errors in this section. For instance, for the RB configuration now it states the following:

https://github.com/TheAxonLab/hcph-sops/blob/10ef2c8a27837d96afe73e170dfdb7fed8b4cbe5/docs/data-collection/preliminary.md#L155

Instead, it should say (i) the specific transducer (TSD160A) because it determines the calibration; and (ii) that setting the transducer determines the specific calibration process (which requires the RB to be fit the same way it will be inside the scanner room).

Then, about the ECG, the following is said:

https://github.com/TheAxonLab/hcph-sops/blob/10ef2c8a27837d96afe73e170dfdb7fed8b4cbe5/docs/data-collection/preliminary.md#L161

Actually, nothing needs revision here if the right parameters are specified.

Finally, regarding the GA:

https://github.com/TheAxonLab/hcph-sops/blob/10ef2c8a27837d96afe73e170dfdb7fed8b4cbe5/docs/data-collection/preliminary.md#L166

This is wrong - it should just have a link to the GA calibration at the end of this same page.

More issues -- something should be said about the visualization ranges, which are not calibration but determine how nicely the signal is represented by AcqKnowldge.

Finally, nothing is said about digital signals.

In summary:

- [ ] Fix the setting of the transducer and calibration of the RB
- [ ] Fix what is said about the calibration and ECG
- [ ] Rewrite the same for the GA and point to the calibration section.
- [ ] Mention configuration of digital signals (including those recorded --all channels-- and visualized --only trigger?--).
- [ ] Mention configuration of visualization ranges
